build_scripts/windows/product.wxs

```diff
@@ -0,0 +1,145 @@
+
+    <?elseif $(var.Platform) = "x86" ?>
+    <?define PromptflowCliRegistryGuid = "7c2c792d-c395-44a1-8222-8e4ea006abb9" ?>
+    <?define PromptflowCliVersionGuid = "f706b208-a15d-4ae7-9185-cfcc43656570" ?>
+    <?define PromptflowSystemPathGuid = "9661fe6a-ff48-4e7c-a60d-fc34c2d06ef3" ?>
+    <?define ProgramFilesFolder = "ProgramFilesFolder" ?>
+    <?define UpgradeCode = $(var.UpgradeCode32) ?>
+    <?define Architecture = "32-bit" ?>
+
+    <?else ?>
+    <?error Unsupported platform "$(var.Platform)" ?>
+    <?endif ?>
+
+    <Product Id="*" Name="$(var.ProductName) ($(var.Architecture))" Language="1033" Version="$(var.ProductVersion)" Manufacturer="$(var.ProductAuthor)" UpgradeCode="$(var.UpgradeCode)">
+        <Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine" />
+
+        <Upgrade Id="$(var.UpgradeCode)">
+          <UpgradeVersion Property="WIX_UPGRADE_DETECTED" Maximum="$(var.ProductVersion)" IncludeMaximum="no" MigrateFeatures="yes" />
+          <UpgradeVersion Property="WIX_DOWNGRADE_DETECTED" Minimum="$(var.ProductVersion)" IncludeMinimum="no" OnlyDetect="yes" />
+        </Upgrade>
+        <InstallExecuteSequence>
+          <RemoveExistingProducts After="InstallExecute" />
+        </InstallExecuteSequence>
+
+
+        <!-- New product architectures should upgrade the original x86 product - even of the same version. -->
+        <?if $(var.UpgradeCode) != $(var.UpgradeCode32) ?>
+        <Upgrade Id="$(var.UpgradeCode32)">
+          <UpgradeVersion Property="WIX_X86_UPGRADE_DETECTED" Maximum="$(var.ProductVersion)" IncludeMaximum="yes" MigrateFeatures="yes" />
+          <UpgradeVersion Property="WIX_X86_DOWNGRADE_DETECTED" Minimum="$(var.ProductVersion)" IncludeMinimum="no" OnlyDetect="yes" />
+        </Upgrade>
+        <Condition Message="A newer version of $(var.ProductName) is already installed.">NOT (WIX_DOWNGRADE_DETECTED OR WIX_X86_DOWNGRADE_DETECTED)</Condition>
+        <?else ?>
+        <Condition Message="A newer version of $(var.ProductName) is already installed.">NOT WIX_DOWNGRADE_DETECTED</Condition>
+        <?endif ?>
+
+        <Media Id="1" Cabinet="promptflow.cab" EmbedCab="yes" CompressionLevel="high" />
+
+        <Icon Id="PromptflowIcon" SourceFile="$(var.ProductResources)logo32.ico" />
+
+        <Property Id="ARPPRODUCTICON" Value="PromptflowIcon" />
+        <Property Id="ARPHELPLINK" Value="https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2" />
+        <Property Id="ARPURLINFOABOUT" Value="https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2" />
+        <Property Id="ARPURLUPDATEINFO" Value="https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2" />
+        <Property Id="MSIFASTINSTALL" Value="7" />
+        <Property Id="ApplicationFolderName" Value="Promptflow" />
```

Comment:
what's this folder? why need to cap the first character?

Reply:
folder when install msi: 
![image](https://github.com/microsoft/promptflow/assets/26239730/4a9caa03-c2c1-4699-995a-b28f69000c8e)

Reply:
have changed to lower case: promptflow